README.md

```diff
@@ -181,6 +181,30 @@ Client.restore(id, :recursive => true)
 client.restore(:recursive => true)
 ```
 
+If you want to validate uniqueness but want to include deleted records:
+
+``` ruby
+class Client < ActiveRecord::Base
+  acts_as_paranoid
+
+  validates :name, uniqueness: { paranoia: :with_deleted }
+
+  ...
+end
+```
+
+If you don't use the default_scope and want to validate uniqueness but want to exclude deleted records:
+
+``` ruby
+class Client < ActiveRecord::Base
+  acts_as_paranoid without_default_scope: true
+
+  validates :name, uniqueness: { paranoia: :without_deleted }
```

Comment:
Don't custom paranoid gem.You can write:
validates :name, uniqueness: {conditions: ->{with_deleted}}

Reply:
> Don't custom paranoid gem.You can write:
> validates :name, uniqueness: {conditions: ->{with_deleted}}

Doesn't work for me

Reply:
Doesn't work for me either in `rails v4.2.10` using MySQL. Traced it a bit and I found that rails wasn't actually removing the scope via the `unscope` method because of the way the relation query is set up. By the time it hits [here](https://github.com/rails/rails/blob/v4.2.10/activerecord/lib/active_record/validations/uniqueness.rb#L29), the relation is wrapped and chained with an `AND` query. It doesn't get through [this case](https://github.com/rails/rails/blob/v4.2.10/activerecord/lib/active_record/relation/query_methods.rb#L912) statement to actually remove the column from the scope because the relation `rel` is a `Arel::Nodes::And` at that point.

I just ended up making my own validator which manually checked for uniqueness against the unscoped records to move forward.